source/Calamari.Aws/Deployment/Conventions/DeployAwsCloudFormationConvention.cs

```diff
@@ -33,734 +27,178 @@ public enum StackStatus
         InProgress
     }
 
-    public class DeployAwsCloudFormationConvention : IInstallConvention
```

Comment:
Holy dumptruck batman ... that was one giant convention :) Good to see this getting more modular